node/test/PackageSourceTests.ts

```diff
@@ -17,6 +17,40 @@ const testSchemaB = "org.opent2t.test.schemas.b";
 const testTranslatorOne = "org.opent2t.test.translators.one";
 const testTranslatorTwo = "org.opent2t.test.translators.two";
 
+const testOnboardingPackageOne = "opent2t-onboarding-org-opent2t-test-onboarding-one";
+const testOnboardingOnboardingOne = "org.opent2t.test.onboarding.one";
+const testOnboardingSchemaOne = "org.opent2t.test.onboarding.one";
```

Comment:
If these 2 are the same then the schema module and onboarding module are the same .js filename, which doesn't work. I don't know what convention was discussed for naming these things.